src/azure-cli/azure/cli/command_modules/serviceconnector/_resource_config.py

```diff
@@ -1104,3 +1104,10 @@ class CLIENT_TYPE(Enum):
 SUPPORTED_CLIENT_TYPE[RESOURCE.ContainerApp] = SUPPORTED_CLIENT_TYPE[RESOURCE.WebApp]
 SUPPORTED_CLIENT_TYPE[RESOURCE.Local] = SUPPORTED_CLIENT_TYPE[RESOURCE.WebApp]
 SUPPORTED_CLIENT_TYPE[RESOURCE.FunctionApp] = SUPPORTED_CLIENT_TYPE[RESOURCE.WebApp]
+
+
+# The dict defines the options for opt-out
+class OPT_OUT_OPTION(Enum):
+    PUBLIC_NETWORK = 'public-network'
```

Comment:
naming convention for argument name is x-x-.., but no need for argument value, how about to remove the hyphen for user convenince,